api/v1/helpers/verbs/doPost.js

```diff
@@ -97,6 +96,10 @@ function doPost(req, res, next, props) {
   }
 
   return postPromise.then((o) => {
+    if (!o) { // could be here from u.handleError
+      return;
```

Comment:
an alternative is to throw err on line 83, instead of u.handleError, which is not a promise and hence postPromise.then returns undefined.